File: Web_API/Web_API/Models/Tree.cs
using System;
using Web_API.Models.Base;

namespace Web_API.Models
{
    public class Tree : BaseModel
    {
        public string Name { get; set; }
        public string Zone { get; set; }
        public string AgeHeight { get; set; }
        public Decimal Price { get; set; }
        public string Description { get; set; }
        //public byte[] Image { get; set; }

        public Guid IdTreeCategory { get; set; }

    }
}

File: vente_arbre_web/src/app/_models/tree/tree.ts
export class Tree {
    public id: string;
    public name: string;
    public zone: string;
    public ageHeight: string;
    public price: string;
    public description: string;
    public image: File;
    public idTreeCategory: string;

    constructor(id: string, name: string, zone: string, ageHeight: string, price: string, description: string, image: File, idTreeCategory: string) 
    { 
        this.id = id;
        this.name = name;
        this.zone = zone;
        this.ageHeight = ageHeight;
        this.price = price;
        this.description = description;
        this.image = image;
        this.idTreeCategory = idTreeCategory;
    }
}